bot/resources/tags/venv.md

```diff
@@ -0,0 +1,21 @@
+**Virtual Environments**
+
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
```

Comment:
I think we can fit a sentence in here explaining why clean dependencies are important. We have an opportunity to make this more beginner friendly without adding too much, and I think we should take it. Eg:
 - knowing what your program actually requires
 - dependency conflicts
 - partial upgrades rather than having to update every script on your system at once

Reply:
Okey doke, totally agree. I'll try to throw some of those in there.